scripts/SANS/isis_reduction_steps.py

```diff
@@ -1066,13 +1074,13 @@ def view(self, instrum):
         instrum.load_empty(wksp_name)
 
         # apply masking to the current detector
-        self.execute(None, wksp_name, instrum)
+        self.execute(None, wksp_name)
```

Comment:
the execute method for this class takes 2 arguments (`reducer` and  `workspace`). Instead of passing through the instrument name, I thought it was more appropriate to pass through the workspace name to the execute method. The instrument is also retrieved the reducer argument in the execute method any way.